Dockerfile.web

```diff
@@ -0,0 +1,46 @@
+###
+## React App
+###
+FROM node:16-bullseye-slim AS nodebuilder
```

Comment:
this image doesn't seem to have yarn installed?

Reply:
yeah it should (ie. tested on my machine)